index.js

```diff
@@ -1,20 +1,45 @@
 const nodePath = require('path');
-const resolve = require('resolve');
+const regexIsCompat = /\.js$/;
+const regexIsRelative = /^\./;
 
 function webify(path) {
-  const specifier = path.node.source.value;
-  const filename = this.file.opts.filename;
-  const base = nodePath.dirname(
-    nodePath.join(process.cwd(), filename)
-  );
-
-  const pth = resolve.sync(specifier, {
-    basedir: base
-  });
-  const rel = nodePath.relative(base, pth);
-  const res = rel[0] === '.' ? rel : `./${rel}`;
-
-  path.node.source.value = res;
+  const { source } = path.node;
+  const { value } = source;
+  const isRelative = regexIsRelative.test(value);
+
+  // If it's web compatible, we simply let it pass through.
+  if (regexIsCompat.test(value)) {
```

Comment:
Yeah, I don't think it's good enough right now. Ideally we could use the original package but I have no idea why it's not working for me. Have you tried?